mindmeld/system_entity_recognizer.py

```diff
@@ -13,15 +13,22 @@
 import json
 import logging
 import sys
-
+import os
 import requests
 
 from mindmeld.components._config import (
     get_system_entity_url_config,
     is_duckling_configured,
 )
+from mindmeld.exceptions import MindMeldError
 
 NO_RESPONSE_CODE = -1
+SYS_ENTITY_REQUEST_TIMEOUT = os.environ.get("MM_SYS_ENTITY_REQUEST_TIMEOUT", 1.0)
+try:
+    if float(SYS_ENTITY_REQUEST_TIMEOUT) <= 0.0:
+        raise MindMeldError("MM_SYS_ENTITY_REQUEST_TIMEOUT env var has to be > 0.0 seconds.")
+except ValueError:
+    raise MindMeldError("MM_SYS_ENTITY_REQUEST_TIMEOUT env var has to be a float value.")
```

Comment:
Can we change this to the following?
```
except ValueError as e:
    raise MindMeldError(...) from e
```
It's a subtle distinction, but it makes the error handling a bit less confusing because it causes python to treat the later exception as a result of the earlier one, rather than an unrelated bug that happened to trigger in the error-handling code.